pyproject.toml

```diff
@@ -37,7 +37,10 @@ mail-parser = "^3.15.0"
 requests = "^2"
 lxml = "^4.9.3" # TODO: remove when https://github.com/adbar/trafilatura/issues/449 is fully resolved
 pillow = "^10.1.0"
-
+pytest = "^7.4.3"
+mongomock = "^4.1.2"
+pyright = "^1.1.343"
+google = "^3.0.0"
```

Comment:
Were these changes intentional? Some of them look like they belong in the `test` poetry group.

Reply:
Pytest, mongomock and pyright should not be there, you are correct!

But for google, I received an error in the testing pipeline for python 3.11 (or 3.12) that the dependency was missing, so I don't know if this should be in the testing dependencies or in the normal ones!